src/extensions/export/bootstrap-table-export.js

```diff
@@ -23,8 +24,7 @@
     $.extend($.fn.bootstrapTable.defaults, {
         showExport: false,
         exportDataType: 'basic', // basic, all, selected
-        // 'json', 'xml', 'png', 'csv', 'txt', 'sql', 'doc', 'excel', 'powerpoint', 'pdf'
-        exportTypes: ['json', 'xml', 'csv', 'txt', 'sql', 'excel'],
+        exportTypes: ['json', 'html', 'xml', 'csv', 'txt', 'sql', 'pdf', 'excel', 'doc', 'powerpoint'],
```

Comment:
@wenzhixin Thats on your decision. @dobobaie rewrote nearly the complete code of the tableExport.js plugin. There are too many changes (325 additions, 1,273 deletions) that prevents me to assess that work and I'm not able to merge dobobaie's pull request with my code base. I suggest that dobobaie continuous his work in his own fork or even shifts it into a complete new project with its own name to become more distinguishable from tableExport.js. dobobaie describes the changes he made with "more light, more clean and easy to dev ext." It's out of my scope to decide if dobobaie's version is suitable to replace my version of the plugin in bootstrap-table. 

Note: This comment should never be a disqualification of dobobaie's work!